### PR TITLE
add more debugging for update_labels, so we can find out why labels

### PR DESCRIPTION
**Pull request by rcerven**
aren't updated properly sometimes

* CLOUDBLD-8615

Signed-off-by: Robert Cerven <rcerven@redhat.com>

# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- n/a JSON/YAML configuration changes are updated in the relevant schema
- n/a Changes to metadata also update the documentation for the metadata
- n/a Pull request has a link to an osbs-docs PR for user documentation updates

### CODE REVIEW

**Review by ben-alkov — APPROVED** [review]

File: osbs/tekton.py [hunk 1]
```diff
@@ -460,6 +463,8 @@ def update_labels(self, labels):
         exc_msg = f"Can't update labels on pipeline run '{self.pipeline_run_name}', " \
                   f"because it doesn't exist"
         response_json = self._check_response(response, msg)
+        logger.debug("labels after update: %s", self.labels)
```
Comment (chmeliik) [inline osbs/tekton.py:466]: Checking the labels before and after is 2 extra HTTP queries on every `update_labels()` call.

**Review by chmeliik — APPROVED** [review]
LGTM assuming this gets removed as soon as it's no longer necessary

**Review by lkolacek — APPROVED** [review]

File: osbs/tekton.py (hunk 1 above)
Comment (rcerven) [reply osbs/tekton.py:466]: yeah but it will be removed, once we figure out root cause for labels not being updated correctly

**Comment by rcerven** [comment]
no rn, just added debug messages

**Pull request merged**